interop-testing/src/main/proto/io/grpc/testing/integration/messages.proto

```diff
@@ -135,6 +135,10 @@ message StreamingOutputCallRequest {
 message StreamingOutputCallResponse {
   // Payload to increase response size.
   Payload payload = 1;
+
+  // Used to track messages produced as a result of onReady callbacks when
+  // outbound flow-control is opened.
+  int32 on_ready_iteration = 2;
```

Comment:
This proto and test.proto are shared with other implementations, so adding this would imply syncing the changes to the other repos, and the change seems java-specific.

I'm not sure what the best alternative is. We could make a new service. Or maybe make the test such that the server shares the values through memory? I think that test could be done pretty easily. And there's no need to put this test in `AbstractTransportTest`; it could be a new test file which would probably make it easier.